frameworks/Crystal/lucky/lucky.dockerfile

```diff
@@ -14,7 +14,7 @@ ENV LUCKY_ENV production
 
 RUN shards build bench --release --no-debug
 
-ENV DATABASE_URL postgres://benchmarkdbuser:benchmarkdbpass@tfb-database:5432/hello_world?initial_pool_size=56&max_idle_pool_size=56
+ENV DATABASE_URL postgres://benchmarkdbuser:benchmarkdbpass@tfb-database:5432/hello_world?initial_pool_size=10&max_idle_pool_size=10
```

Comment:
Sorry, only an small question !!
All are playing with different pool sizes for the databases.

But we are learning (all languages) witch one is better ???

Reply:
That's only a question for reflexion !!
So we can learn all together.

Nothing about this PR.

Reply:
No problem @joanhey  😄 
I think the pool size is already per CPU core.
This was causing the runs to fail as it was making too many connections.